### PR TITLE
Add nginx ingress paths for the prometheus versions being benchmarked

### LINKED ISSUE #164: expose master and pr prometheus instances by default so we can run custom queries
in most cases so far it would be usefull to run custom quiries 

for example in https://github.com/prometheus/prometheus/pull/4768 and https://github.com/prometheus/prometheus/pull/4763

### PR DESCRIPTION
**Pull request by sipian**
Closes #164 

This exposes the Prometheus version on the following URLs

```
http://prombench.prometheus.io/{{ .PR_NUMBER }}/prometheus-pr
http://prombench.prometheus.io/{{ .PR_NUMBER }}/prometheus-release
```
I will also add these links in the welcome message of prombot.

Signed-off-by: sipian <cs15btech11019@iith.ac.in>

### CODE REVIEW

**Comment by sipian** [comment]
I have tested this successfully on the cluster.

**Comment by krasi-georgiev** [comment]
can you also add the links of the exported PR and RELEASE Prometheus to  the comment sent to github when starting the benchmark.

File: components/prombench/manifests/results/prometheus-meta.yaml [hunk 1]
```diff
@@ -89,8 +89,18 @@ data:
...
+      - action: replace
+        source_labels: [__meta_kubernetes_namespace,__meta_kubernetes_service_label_prometheus]
+        regex: prombench-(\d+);test-pr-\d+
+        target_label: __metrics_path__
+        replacement: /${1}/prometheus-pr/metrics
+      - action: replace
+        source_labels: [__meta_kubernetes_namespace,__meta_kubernetes_service_label_prometheus]
+        regex: prombench-(\d+);test-(?:master|v.+)
+        target_label: __metrics_path__
+        replacement: /${1}/prometheus-release/metrics
```
[the remark this answers is not in the document]
Comment (sipian) [reply components/prombench/manifests/results/prometheus-meta.yaml]: Due to web-external-url in Prometheus being benchmarked, metrics endpint will be exposed on a different path.
We need to replace the default path from `/metrics` to the exposed URL path

File: temp/prometheus-builder/build.sh [hunk 2]
```diff
@@ -25,4 +25,5 @@ printf "\n\n>> Starting prometheus\n\n"
 ./prometheus --config.file=/etc/prometheus/config/prometheus.yaml \
              --storage.tsdb.path=/data \
              --web.console.libraries=${DIR}/console_libraries \
-             --web.console.templates=${DIR}/consoles
\ No newline at end of file
+             --web.console.templates=${DIR}/consoles \
+	       --web.external-url=http://prombench.prometheus.io/$PR_NUMBER/prometheus-pr
```
[the remark this answers is not in the document]
Comment (sipian) [reply temp/prometheus-builder/build.sh]: Aaah I'll fix it. It was aligned correctly in my text-editor

**Comment by sipian** [comment]
I have opened a PR in test-infra with the updated welcome message

**Comment by krasi-georgiev** [comment]
Lgtm.
Please push the new image after merging this.
Thanks

**Pull request merged**